owsi-core/owsi-core-components/owsi-core-component-jpa-more/src/main/java/fr/openwide/core/jpa/more/business/history/model/atomic/HistorySubjectOrganization.java

```diff
@@ -0,0 +1,7 @@
+package fr.openwide.core.jpa.more.business.history.model.atomic;
+
+public enum HistorySubjectOrganization {
+	
+	FAFCEA
```

Comment:
Probably a leftover from the original project. Should be removed, or moved to the basic application (maybe we can introduce an "organization" property in the archetype, so that we know how to name this value?).
The point of this class is to populate the "subjectOrganization" property of HistoryLog when the user is internal and when there's no business model for the internal organization.